components/camel-dozer/pom.xml

```diff
@@ -28,7 +28,7 @@
 
     <artifactId>camel-dozer</artifactId>
     <packaging>jar</packaging>
-    <name>Camel :: Dozer</name>
+    <name>Camel :: Dozer (Deprecated)</name>
```

Comment:
It should be in lower-case, then we have tooling that detects this and mark it officially as deprecated.

Reply:
Ahhhh, nice. I had seem 2 classes previously marked as deprecated so I assumed we had done that in the past. 

I'll adjust. Thanks for the heads up.